### ISSUE TITLE
爱可可微博热门分享(1.19)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#393]
No 1. <a href="https://weibo.com/1402400261/PaaFf5ni5">Cursor的11个实战技巧 - 从项目规划到代码优化</a>
No 2. <a href="https://weibo.com/1402400261/PaiDGCZfS">LeetGPU：无需GPU即可免费编写和运行CUDA代码的平台</a>
No 3. <a href="https://weibo.com/1402400261/PaiNAopY1">《你想知道的关于数学的一切》是一本探讨数学思维、解决问题和写作证明的书籍，旨在帮助读者理解抽象数学，并通过一系列的例子和练习引导读者学会如何进行数学证明</a>
No 4. <a href="https://weibo.com/1402400261/PaaBDp5FL">免费书稿《爱丽丝梦游可微仙境：神经网络设计基础》，通过探索可微的数学基础，旨在帮助读者理解和设计深度神经网络</a>
No 5. <a href="https://weibo.com/1402400261/P9Z2nC9oi">《思考，快与慢》</a>
No 6. <a href="https://weibo.com/1402400261/PaixyzwkG">论文Discussion部分怎么写？从重要性到未来展望</a>
No 7. <a href="https://weibo.com/1402400261/Paffu5ggI">[LG] Towards Large Reasoning Models: A Survey of Reinforced ......</a>
No 8. <a href="https://weibo.com/1402400261/PabLQ8AG3">人们常说，要追求卓越，创造奇迹。但生活的真相是：一盏始终如一的烛光，往往比偶尔闪现的烟花更有价值。 稳定的付出，就像永不熄灭的蜡烛，虽然光芒不够耀眼，却能持续照亮前方的道路。而那些昙花一现的精彩表现，就像绚丽的烟火，虽然令人惊叹，却转瞬即逝。 ......</a>
No 9. <a href="https://weibo.com/1402400261/P9YCJ2CuF">[149星]TorchLeet：PyTorch的练习题库，就像编程界的“题海战术”，通过各种难度的实践题目，帮助你从新手到高手，掌握深度学习和PyTorch的精髓</a>
No 10. <a href="https://weibo.com/1402400261/Paht0oXZy">[176星]note-gen：一款跨端AI笔记应用...</a>
No 11. <a href="https://weibo.com/1402400261/P9O0Fe8q4">「2025.1版AI Agent工具全景图： 从入门到进阶的12个关键工具」 1. 入门首选： PydanticAI ......</a>
No 12. <a href="https://weibo.com/1402400261/Pafa6z3F2">[CL]《Imagine while Reasoning in Space: Multimodal Visualization-of-Thought》</a>
No 13. <a href="https://weibo.com/1402400261/Pa5FT36fm">[LG] Agentic Retrieval-Augmented Generation: A Survey on Agentic ......</a>
No 14. <a href="https://weibo.com/1402400261/PaiPch3du">如何构建一个由多个专业化 AI Agent 组成的游戏设计团队，以帮助开发者自动化生成游戏概念，确保游戏叙事、游戏玩法、视觉和技术规格协同工作</a>
No 15. <a href="https://weibo.com/1402400261/PaeShk6zo">早！[太阳] ​</a>
No 16. <a href="https://weibo.com/1402400261/P9Xe9F38m">Hugging face关于如何构建AI Agent的课程，介绍了 AI Agent 的概念、构建高效Agent的关键组件、以及通过 smolagents 库实现这些代理的方法</a>
No 17. <a href="https://weibo.com/1402400261/PahltrSjJ">[20星]Compose-RL：一个用于强化学习与人工反馈（RLHF）的框架，旨在简化不同 RLHF 技术的集成，提供模块化和组合式的实验能力，适用于研究人员和实践者</a>
No 18. <a href="https://weibo.com/1402400261/PaaoOdUUG">[724星]openai-realtime-agents：展示如何在实时 API 之上构建更高级、具有Agent式模式的简单演示项目</a>
No 19. <a href="https://weibo.com/1402400261/PaaqUFjJM">[36星]Awesome-Human-Motion：关于人类运动理解和生成研究的聚合项目，包含了大量的学术研究、数据集、以及人类运动分析和合成的相关技术和方法</a>
No 20. <a href="https://weibo.com/1402400261/Paaq8547c">[307星]XiYan-SQL：针对自然语言转 SQL 任务的多生成器集成框架，旨在提高大型语言模型在生成 SQL 查询方面的性能</a>
No 21. <a href="https://weibo.com/1402400261/PahpK5Wah">[198星]100-languages：用100种不同编程语言解100个数学难题，探索编程语言的多样性和解决问题的创新方式</a>
No 22. <a href="https://weibo.com/1402400261/P9Z5DrvcO">[1.5k星]AI-Crash-Course：人工智能速成课程，旨在帮助忙碌的建设者在短短两周内赶上人工智能研究的前沿。它提供了一系列精选的资源，包括视频、论文和网站，帮助学习者快速了解AI的基础知识和最新进展。</a>
No 23. <a href="https://weibo.com/1402400261/PahnheaJB">[156星]llmpeg：使用大型语言模型（LLM）来生成 ffmpeg 命令的工具，让视频处理变得简单，通过自然语言生成复杂的ffmpeg命令，轻松完成视频音频编辑，无需专业技术</a>
No 24. <a href="https://weibo.com/1402400261/P9YW2bu6w">[140星]Memobase：为生成式AI应用打造的用户画像记忆系统，能帮助AI记住用户的各种信息，提升用户体验和留存率</a>
No 25. <a href="https://weibo.com/1402400261/PaiKEaX8E">ITOA：将图片转换为 ASCII 艺术的在线工具，拖放图片到指定区域进行转换</a>
No 26. <a href="https://weibo.com/1402400261/Paaxjw7ip">[1.6k星]React Bits：一个开源库，提供多种动画、交互式且完全可定制的 React 组件，旨在帮助开发者构建吸引人的用户界面</a>
No 27. <a href="https://weibo.com/1402400261/Paawp0t2r">[338星]Steam Insights：包含 Steam 视频游戏数据的数据库，数据截止至 2024 年 10 月，涵盖游戏详情、类型、评论、标签以及 SteamSpy 的洞察分析，对于游戏开发者和爱好者来说，是个了解市场和寻找灵感的好帮手</a>
No 28. <a href="https://weibo.com/1402400261/Pag4ScGXu">通俗版解读</a>
No 29. <a href="https://weibo.com/1402400261/P9Ode0p9I">Build an AI Teaching Agen...</a>
No 30. <a href="https://weibo.com/1402400261/PaiD5eazK">“What’s Next for Mamba? Towards More Expressive Recurrent Update ......</a>
No 31. <a href="https://weibo.com/1402400261/Paf333pty">[CL]《The Lessons of Developing Process Reward Models in Mathematical Reasoning》</a>
No 32. <a href="https://weibo.com/1402400261/P9IMalMEB">前端展现层</a>
No 33. <a href="https://weibo.com/1402400261/P9NM3EHHm">AI编程蓝皮书：一份面向AI编程新手的实用手册，详细介绍了如何使用Cursor、Windsurf和Coze等工具开发AI产品，从基础操作到实战案例，助力零基础者快速上手AI编程，开启创造之旅</a>
No 34. <a href="https://weibo.com/1402400261/P7ZEvpZVS">一份值得参考的2025年AI书单1. 《AI Engineering》</a>
No 35. <a href="https://weibo.com/1402400261/PaapNf3Zq">[4k星]Scira：由 Vercel AI SDK 提供支持的、以 Grok 2.0 等模型为搜索引擎的最小化 AI 搜索引擎项目</a>
No 36. <a href="https://weibo.com/1402400261/PabZ9g24l">晚安～ [月亮] ​</a>
No 37. <a href="https://weibo.com/1402400261/Paf5CwHAH">[LG]《Tensor Product Attention Is All You Need》</a>
No 38. <a href="https://weibo.com/1402400261/Pahqiu8hc">[43星]ScribeAgent：基于大规模生产工作流数据训练的智能网络Agent，可将开源大型语言模型微调为专门的网络Agent，帮助用户更高效地完成网络任务</a>
No 39. <a href="https://weibo.com/1402400261/Pafo0jZXD">今日推介(第1655期)：从离散时间策略到连续时间扩散采样器、面向稳定大语言模型训练的尖峰感知Adam优化器与动量重置、开发数学推理中过程奖励模型的经验教训、张量积注意力机制、多模态思维可视化 ......</a>
No 40. <a href="https://weibo.com/1402400261/P9YFQ6CNM">[4.1k星]PyTorch Forecasting：基于PyTorch的时间序列预测工具，它能帮助用户轻松实现高精度的时间序列预测，无论是专业人士还是初学者都能快速上手，为实际应用和研究提供强大支持</a>
No 41. <a href="https://weibo.com/1402400261/P9RpVChWY">[138星]Awesome-CLIP：基于 CLIP（Contrastive Language-Image Pretraining）的研究成果汇总项目，旨在整理和分享 CLIP 相关的研究进展，包括 CLIP 的训练方法、改进、数据处理、知识蒸馏、损失函数、以及在零样本、少样本和分类等任务中的应用</a>
No 42. <a href="https://weibo.com/1402400261/PahpbnGZc">[38星]Physics-IQ: 一个帮助AI理解物理世界的视频基准数据集，通过真实视频评估生成模型的物理知识，助力AI更好地把握自然规律</a>
No 43. <a href="https://weibo.com/1402400261/P9NLjrfh9">2024下半年值得关注的AI论文：涵盖从专家模型到LLM精度扩展法则的六篇有影响力的AI论文，为AI领域从业者和研究人员提供了前沿的研究进展和趋势</a>
No 44. <a href="https://weibo.com/1402400261/Pa9lZF3UM">构建基于生成式 AI 的应用时常见的陷阱，包括不必要地使用 AI、混淆产品问题与 AI 问题、过于复杂的初始设计、对早期成功过度依赖、忽视人工评估以及众包确定应用场景等</a>
No 45. <a href="https://weibo.com/1402400261/P9YVlcWx5">[25星]FloodCast：大规模洪水建模与预测工具，能快速、准确地预测洪水范围和深度，为防洪减灾提供有力支持</a>
No 46. <a href="https://weibo.com/1402400261/P9YyZkjAu">[503星]OpenScholar：这是一个助力科研人员的智能工具，能从海量科学文献中快速找到相关信息并生成回答，就像给科研工作配了个“智能助手”，帮助节省时间、提升效率</a>
No 47. <a href="https://weibo.com/1402400261/P9YTSBxWS">[60星]awesome-lifelong-llm-agent：关于基于大型语言模型（LLM）的终身学习（Lifelong Learning）代理（Agent）的资源和研究论文的库</a>
No 48. <a href="https://weibo.com/1402400261/Pag2ZxNGL">通俗版解读 查看图片//[LG]《SPAM: Spike-Aware Adam with Momentum Reset for Stable LLM Training》</a>
No 49. <a href="https://weibo.com/1402400261/Pag1VCY9K">通俗版解读 查看图片//[LG]《From discrete-time policies to continuous-time diffusion samplers: Asymptotic equivalences and faster training》</a>
No 50. <a href="https://weibo.com/1402400261/Pag4e1ym6">通俗版解读 查看图片//[LG]《Tensor Product Attention Is All You Need》</a>
